change/@internal-react-composites-7fa0abbc-d304-4a7d-9889-bf8d1677770c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added logic to meeting composite to store unread messages",
```

Comment:
nitty nit: Could be terse, and a bit more accurate "MeetingComposite: store messages since last time ChatPane was opened"

"unread" has a different meaning, being pedantic, and we're only approximating that.